javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,47 @@
+const width = 19;
+const height = 29;
+const area = width * height;
+console.log('value of area: ', area);
+console.log('typeof area: ', typeof area);
+
+const bill = 23;
+const payment = 52;
+const change = payment - bill;
+console.log('value of change: ', change);
+console.log('typeof change: ', typeof change);
+
+const quizzes = 10;
+const midterm = 80;
+const final = 85;
+const grade = (quizzes + midterm + final) / 3;
+console.log('value of grade: ', grade);
+console.log('typeof grade: ', typeof grade);
+
+const firstName = 'Matt';
+const lastName = 'Kirby';
+const fullName = firstName + ' ' + lastName;
+console.log('value of fullName: ', fullName);
+console.log('typeof fullName: ', typeof fullName);
+
+const pH = 19;
+let isAcidic;
+if (pH < 7) {
+  isAcidic = true;
+} else {
+  isAcidic = false;
+}console.log('value of isAcidic: ', isAcidic);
+console.log('typeof isAcidic: ', typeof isAcidic);
+
+const headCount = 9;
+let isSparta;
+if (headCount === 300) {
```

Comment:
using an `if` statement like this totally works, but since we are using a comparison operator (`===`), the result will evaluate to a boolean.

You can do something like this and get the same results:
`const isSparta = headCount === 300;`